### PR TITLE
Football project

### PR DESCRIPTION
**Pull request by nejidevelops**
### I ran out of API requests on the previous PR and so I had to create a new pull request based on a new API project

In this PR:
- I have fully implemented the app and it is functional.
- I have also worked on the tests

Here is a screenshot of the tests

Screenshot from 2023-09-08 10-57-25

***Feel free to give a feedback and I'll take it with a smile :blush:***

### CODE REVIEW

**Review by AbelG101 — COMMENTED** [review]
# Hi @mohammadnewton 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

- Hello @mohammadnewton, since this is a rereview you'll have to make the changes on the previous PR instead of creating a new one and make sure to leave a note so future reviewers can understand the situation.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by Nemwel-Boniface — COMMENTED** [review]
## Hello @mohammadnewton   👋,

I am Nemwel your reviewer.


## _This is an additional review as a result of the second opinion request._

I have received your second opinion request with a complaint about the previous reviewer @AbelG101 . Below are my findings:

While your reasons for creating a new PR are actually valid, kindly note that we also have set standards and protocols that are followed to ensure that there is consistency across all the reviews that are provided by the reviewers at Microverse. @AbelG101  had a hard choice of proceeding with your review request or invalidating and he correctly chose to invalidate. The situation here is, that he reviewed your project, and another student with an exactly similar case gets his/ her project invalidated. Do you see where consistency comes into play?

**What you need to do:**

- [ ] Just as @AbelG101  suggested, you need to use the old branch that you used to create the first PR for review to make the new changes that you have made in this PR. You have a few options including: 

1. Creating a PR and merging all the changes that are in this branch into the other old branch and still using that old PR for review and it will be accepted.
2. Re-add all the changes you made here to the old branch and still use the same old PR for requesting review.

Ultimately, the choice is yours and you are the one to choose which option resonates best to you.


In case anything is not 100% clear tag me @Nemwel-Boniface so that I get the notification.

Keep rocking 🚀!

**Comment by enniopo** [comment]
To the next CR: **As an exception**, please proceed to review the PR. 
Do your review as if it were a first review, and then open a zendesk ticket so we can record your review as the first review also on our database.

File: README.md [hunk 1]
```diff
@@ -54,6 +54,18 @@ This is the capstone project for react. It consumes an API, uses Redux as a stat
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
+### Video Demo <a name="video-demo"></a>
+
+[Video Link](https://www.loom.com/share/36ccb36709224b2ba879bc5bfc76cb63?sid=d50db147-b8c9-44eb-bf1d-f2bd069e3512)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (mikethreels) [inline README.md:61]: - [ ] very nice demo video, You present the application very clearly, but according to the project requirements, you should also include a highlight about a piece of the code you find interesting or are proud of making.

File: README.md [hunk 2]
```diff
@@ -114,7 +126,7 @@ You can deploy this project using:
 
 - GitHub: [@newtonOmbese](https://github.com/mohammadnewton)
 - Twitter: [@NewtonOmbese](https://twitter.com/mohanewto200)
-- LinkedIn: [@newtonOmbese](https://www.linkedin.com/in/newton-ombese-570862210/)
+- LinkedIn: [@newtonOmbese](https://www.linkedin.com/in/mohammad-newton-ombese-570862210/)
```
Comment (mikethreels) [inline README.md:127]: - [ ] according to the requirements The README should contain an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/).

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Newton, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Good job with the react Carousel
- very nice adding the read more/read less functionality

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ichala — APPROVED** [review]
Hi Student  ,
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:  
 
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

# To Highlight 💪🏻

- Readme Updated✔️ 
- I can see in your video that you talked about some extra features that you implemented so you spoke about pieces of codes that you are proud of indirectly, however, kindly make sure the next time to be direct and say that you are proud of those features 🙏 ✔️


 Good job implementing the requirements  :+1: Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Optional Suggestions  :

- N/A


**********
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  ichala in your question so I can receive the notification.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**